mlflow/artifacts/__init__.py

```diff
@@ -10,8 +10,10 @@
 from mlflow.protos.databricks_pb2 import INVALID_PARAMETER_VALUE, BAD_REQUEST
 from mlflow.tracking import _get_store
 from mlflow.tracking.artifact_utils import _download_artifact_from_uri, get_artifact_repository
+from mlflow.utils.annotations import developer_stable
 
 
+@developer_stable
```

Comment:
What should be decorated by `developer_stable`? All public APIs/classes?

Reply:
Even some internal ones that we feel would be interfaced with by integrations for which we can pledge to not modify the signature or general behavior from an API standpoint.